fairlearn/metrics/_annotated_metric_function.py

```diff
@@ -25,7 +25,13 @@ class AnnotatedMetricFunction:
     `DataFrame.apply()`. The net result of this is that it is useful
     to have a function wrapper which can be invoked with a DataFrame
     argument, and extract its arguments from that DataFrame.
-
+    .. versionadded:: 0.7
+            In previous versions only the ``predict`` method was used
+            implicitly.
+    .. versionchanged:: 0.7
+            From version 0.7, 'predict' is deprecated and the default will
+            change to 'auto' from v0.10.
+  
```

Comment:
@eskayML thanks for finding all the locations where we need to add `versionadded` and `versionchanged`. They need to actually match the class/function, though. So it's about cross-checking with the past versions to see where it was added, and replace 0.7 with that version. If there were no major changes to it since then, `versionchanged` is not needed. If there were major changes (e.g., to the API) then the version in which it changed should be named instead of 0.7 and a description of the change should be added.

Does that make sense?

Reply:
Interesting, so for example the `[_annotated_metric_function.py](https://github.com/fairlearn/fairlearn/blob/release/v0.8.X/fairlearn/metrics/_annotated_metric_function.py)` which looks to have been created in version 8, I'm to use just  the version added alone 

Also I'd love to know if i'm to add the description under the `versionadded` or `versionchanged` just like the example you gave in #855 , I think that might be a little challenging for me.

Reply:
Thanks for clarifying it all.